shlifim/settings.py

```diff
@@ -70,6 +70,14 @@
     },
 ]
 
+TEMPLATE_CONTEXT_PROCESSORS = (
+    "django.core.context_processors.auth",
+    "django.core.context_processors.debug",
+    "django.core.context_processors.i18n",
+    "django.core.context_processors.media",
```

Comment:
what is "TEMPLATE_CONTEXT_PROCESSORS " used for? I can't find an explanation online.

Reply:
the first 4 lines are Django default. 
the 5th `django.core.context_processors.request`
create the ability to every request to know the previous path, to be able to return to the previous page after login.